### ISSUE TITLE
Randomize command on selected cells

### ISSUE THREAD

**Issue by michaelsjackson** [JoepVanlier/Hackey-Trackey#49]
Select a few cells, trigger randomize. shift-r could be the key binding, as ctrl-r is already used for record.

**Comment by JoepVanlier** [comment]
Randomize what exactly? Between what ranges? What distribution is used in the randomization? I don't really want to add a bunch of UI controls for determining ranges and custom options for the randomization and without those I don't see how this would be that useful. You would rarely randomize over the full range over a value.

**Comment by michaelsjackson** [comment]
randomizing the selected values in the range min..max, usually 00..7F or 00..FF, depending on the cell. Sometimes it is good not to think if it makes any sense or not, it would be just another feature, for potential (ab)use.

**Comment by JoepVanlier** [comment]
I'm not a fan of bloating with features that are not useful. Not only does every feature need to be maintained, the help-file is also getting increasingly unreadable with all the extra key bindings and the chance of calling a command one didn't intend also increases with more and more keys being bound.

**Comment by michaelsjackson** [comment]
I agree fully, this is a feature, we can forget about easily, where we have anyway the shifting solution, which is like random, in a more controled fashion.
